opentracing-shim/src/main/java/openconsensus/opentracingshim/SpanShim.java

```diff
@@ -74,25 +75,28 @@ public Span setTag(String key, Number value) {
 
   @Override
   public Span log(Map<String, ?> fields) {
-    // TODO
+    // TODO - verify 'null' for 'event' is valid.
+    span.addEvent(null, convertToAttributes(fields));
     return this;
   }
 
   @Override
   public Span log(long timestampMicroseconds, Map<String, ?> fields) {
-    // TODO
+    // TODO - use timestampMicroseconds
+    span.addEvent(null, convertToAttributes(fields));
```

Comment:
Here should be special treatment for the attributes `event` and `message`. Those can be used as a name

Reply:
Even though some `Tracer`s and APIs do that, nobody is actually forced to provide such `message` nor `event` name. We need to think how to handle this (including the case when neither of them was passed).

Reply:
during the meeting we decided to extract `event` from the fields. In all other cases use `log` as a name